utils/prometheus-node-exporter-lua/files/etc/init.d/prometheus-node-exporter-lua

```diff
@@ -32,13 +34,35 @@ start_service() {
 	procd_set_param command /usr/sbin/uhttpd -f -c /dev/null -l / -L /usr/bin/prometheus-node-exporter-lua
 	[ $keepalive -gt 0 ] && procd_append_param command -k $keepalive
 
-	if [ "$interface" = "*" ]; then
+	if [ "$interface" = "*" ] && [ "$cert_path" = "none"] && ["$key_path" = "none"]; then
 		procd_append_param command -p $port
-	else
+	elif [ "$cert_path" = "none"] && ["$key_path" = "none"]; then 
 		[ -n "$bind4" ] && procd_append_param command -p $bind4:$port
 		[ -n "$bind6" ] && procd_append_param command -p [$bind6]:$port
 	fi
```

Comment:
What if only one of the params `$cert_path` or `key_path` is not equal `none`
?

Reply:
The condition shall ensure, that both $cert_path and $key_path are set, since both are required for TLS. Essentially, since "none" is the default value, I did not expect this condition to fail. But maybe this is a wrong assumption and it would be better to check for each value individually. I'd appreciate your feedback on this.

Reply:
Fixed your suggestion in the latest commit to the branch